client/core/core.go

```diff
@@ -3202,6 +3218,9 @@ func (c *Core) resumeTrades(dc *dexConnection, trackers []*trackedTrade) assetMa
 				notifyErr("Order coin error", "Source coins retrieval error for %s %s: %v", unbip(wallets.fromAsset.ID), tracker.token(), err)
 				continue
 			}
+			// NOTE: change and changeLocked are not set even if the funding
+			// coins were loaded from the DB's ChangeCoin.
```

Comment:
Good point. We should look into doing this a little more carefully, if not just for accounting purposes. Right now we're not displaying the current or previous change coins anywhere, so it's of little consequence, but still a little sloppy.